simulators/nav/src/utils/types.ts

```diff
@@ -128,12 +128,20 @@ export interface Obstacle {
   size:number; /* meters */
 }
 
+/* Type representing the Obstacle LCM. This must be the same as the
+  Obstacle LCM. +x is to the right of camera, +y is down from camera,
+  +z is forward (out) from the camera */
+export interface ObstacleNewInterfaceMessage {
+  bottom_left_coordinate_meters:number[];
+  top_right_corrdinate_meters:number[];
+}
 
 /* Interface representing the Obstacle LCM. This must be the same as the
    Obstacle LCM. */
 export interface ObstacleMessage {
```

Comment:
Do we need this anymore? Or are you keeping it in for reference? (might want to but a deprecated note if you are)

Reply:
Because I did not want to mess up with how the current 2d sim detects obstacles within the FOV, I basically did a conversion of the old obstacle message to the new one and calculate the bounding box using the size of the obstacles